samples/net/download/prj.conf

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+
+CONFIG_DOWNLOAD_CLIENT=y
+CONFIG_DOWNLOAD_CLIENT_STACK_SIZE=4096
+
+CONFIG_NEWLIB_LIBC=y
+
+# Networking
+CONFIG_NETWORKING=y
+# CONFIG_NET_SOCKETS_OFFLOAD=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_TCP=y
+CONFIG_NET_NATIVE=y
+CONFIG_HEAP_MEM_POOL_SIZE=1024
+CONFIG_MAIN_STACK_SIZE=8192
```

Comment:
Do we need 8k?

Reply:
No, but need 4k for wifi, for lte the default is good enough. Would it be best to remove the MAIN_STACK_SIZE config here and just set it in the board config?